index.html

```diff
@@ -51,5 +51,201 @@ <h3 class="connect">LET'S CONNECT</h3>
+          <li class="light">2015</li>
+        </ul>
+        <p class="description-2 flex0">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="ul-box2 flex0">
+          <li class="highlight font-1">html</li>
+          <li class="highlight font-2">css</li>
+          <li class="highlight font-3">javaScript</li>
+        </ul>
+        <a href="#" class="highlight-2 font-4 flex0"> See Project </a>
+      </div>
+    </section>
+    <section class="about-myself">
+      <div class="container-3">
+        <div class="top">
+          <h1 class="first-title-2">About Myself</h1>
+          <p class="description-3">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h3 class="connect-2">LET'S CONNECT</h3>
+          <ul class="container-2">
+            <li>
+              <a href="#"> <img src="images/twiter.svg" alt="Twitter" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/Linkedin.svg" alt="Linkedin" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/m.svg" alt="m" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/github.svg" alt="github" /></a>
+            </li>
+            <li>
+              <a href="#"> <img src="images/manito.svg" alt="manito" /></a>
+            </li>
+          </ul>
+          <a href="#" class="highlight-3 font-5"> Get my resume </a>
+        </div>
+        <div class="skill-list">
+          <ul class="ul-box3">
+            <li class="li-1">
+              <div class="flex1">
+                <div class="lang">Langguages</div>
+                <div class="arrow">
+                  <img
+                    src="images/downarrow.png"
+                    alt="downarrow"
+                    width="14"
+                    height="8"
+                  />
+                </div>
+              </div>
+              <div class="frame grid">
+                <div>
+                  <img src="images/Ellipse-1.png" class="js" alt="javaScript" />
+                </div>
+                <div><p class="langu">JavaScript</p></div>
+              </div>
+              <div class="frame grid">
+                <div>
+                  <img src="images/Ellipse-2.png" class="js" alt="javaScript" />
+                </div>
+                <div><p class="langu">HTML</p></div>
+              </div>
+              <div class="frame grid">
+                <div>
+                  <img src="images/Ellipse-3.png" class="js" alt="javaScript" />
+                </div>
+                <div><p class="langu">CSS</p></div>
```

Comment:
- [ ] Let's make this code more semantic instead of using `div`, make "JavasScript", "HTML" and "CSS" a list `ul` and each of them a `li`